### ISSUE TITLE
zmbackup version 1.2.3 not working

### ISSUE THREAD

**Issue by sargishindoyan** [lucascbeyeler/zmbackup#150]
Bug Report
##### ISSUE TYPE
- Bug Report

##### ENVIRONMENT VERSION
- Zmbackup Version: 1.2.3
- Zimbra Version: 8.8.15
- Linux Distribution & Version: centos 7

##### SUMMARY
install.sh doesn't work correctly if the command zmbackup -v is not working. Should be change the check to which now.

##### STEPS TO REPRODUCE
```run ./install.sh
su zimbra 
zmbackup -v


```

##### ACTUAL RESULTS
```
```bash comand not found
![Screenshot from 2020-09-25 20-16-27](https://user-images.githubusercontent.com/71894338/94290965-03970080-ff6c-11ea-9d14-d7521bb3b681.png)

**Comment by lucascbeyeler** [comment]
Hi @sargishindoyan 

Can't promisse working on this bugfix as I'm working on Waddles at the moment.

**Comment by lucascbeyeler** [comment]
Fixed in 1.2.5 - please try again with that version
